dockerfiles/hpcc/templates/_helpers.tpl

```diff
@@ -47,9 +47,25 @@ Generate global ConfigMap info
 Pass in root as .
 */}}
 {{- define "hpcc.generateGlobalConfigMap" -}}
+{{- $local := dict "defaultEsp" "" -}}
 imageVersion: {{ required "Please specify .global.image.version" .Values.global.image.version | quote }}
 singleNode: {{ .Values.global.singleNode | default false }}
-defaultEsp: {{ .Values.global.defaultEsp | default (index .Values.esp 0).name }}
+
+{{-/* Set the globalEsp to whatever was set globally, or the first non-disabled esp (if there is one) */-}}
```

Comment:
I think needs a space after/before '-'

Reply:
@richardkchapman - I ran it and got 'parse error at (hpcc/templates/_helpers.tpl:54): "-"